presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -331,6 +331,14 @@ void PrestoServer::run() {
           proxygen::ResponseHandler* downstream) {
         server->reportMemoryInfo(downstream);
       });
+  httpServer_->registerPut(
```

Comment:
Why is the `v1/memory` endpoint overloaded? Might be better to add a new endpoint `v1/memory/clear`